app/helpers/dashboard_helper.rb

```diff
@@ -117,4 +117,19 @@ def report_color
       :disabled_hosts => "#92A8CD"
     }
   end
+
+  def welcome_links
+    [
+      [_("Create Host"), hash_for_new_host_path, :class => 'btn btn-success'],
+      [_("Discovered Hosts"), hash_for_new_host_path, :class => 'btn btn-default']
```

Comment:
This would get moved to the plugin, and discovery can extend this method with alias_method_chain.